### ISSUE TITLE
Error loading SPRT chunk

### ISSUE THREAD

**Issue by Jdewi12** [krzys-h/UndertaleModTool#1082]
### Describe the bug

> An error occured while trying to load:
> Expected 1
> at 00025190 while reading object
> UndertaleModLib.Models.UndertaleSprite
> while reading item 76 of 385 in a list of
> UndertaleModLib.Models.UndertaleSprite in chunk SPRT

(I typed this by hand since I can't seem to find a log file)

### Reproducing steps

Open [Automon](https://store.steampowered.com/app/1822710/Automon/)'s data.win

### Setup Details

UndertaleModTool version: 0.4.04 and the Bleeding edge for commit https://github.com/krzys-h/UndertaleModTool/commit/856041c5279caa9d0ecd5188b8dff7c0e6968d17. Both gave same error.
GameMaker Game: [Automon](https://store.steampowered.com/app/1822710/Automon/)

Let me know if there's something in the data.win you want me to check/share. I'm pretty comfortable with a hex editor.

**Comment by Jdewi12** [comment]
On a sidenote, I couldn't figure out how to get a GameMaker version number. I tried using the info at https://pcy.ulyssis.be/undertale/unpacking-corrected.

The Major.Minor.Release.Build from gen8 are 2.0.0.0, but I doubt that's right.
The InfoFlags are simply 0x32 -- no version info.

The "unknown" int24 at the start of gen8 is 17, in case that's the bytecode version. Though I imagine it's not.

**Comment by Jacky720** [comment]
How many of the following chunks does it have? TGIN (2.2.1), SEQN (2.3), FEDS (2.3.2), FEAT (2022.8).

**Comment by Jdewi12** [comment]
TGIN: 1
SEQN: 1
FEDS: None
FEAT: None

**Comment by Jacky720** [comment]
Yep. Seems like the crash might have to do with nineslice sprites, but it's hard to tell without running the tool in debug mode.

**Comment by Jacky720** [comment]
@Jdewi12 Is this issue still present on 0.5.1.0 or current bleeding edge?

**Comment by Jdewi12** [comment]
It still happens on both 0.5.1.0 and bleeding edge with the version of data.win from when I created this issue, but it doesn't happen on the latest version of the game. 

It's not an issue for Automon anymore, but if you do want to look into it further, I've attached the old data.win file (zipped).

[data.zip](https://github.com/UnderminersTeam/UndertaleModTool/files/14428839/data.zip)
